src/Form/Control/Password.php

```diff
@@ -4,7 +4,43 @@
 
 namespace Atk4\Ui\Form\Control;
 
+use Atk4\Ui\Button;
+use Atk4\Ui\Js\JsExpression;
+
 class Password extends Line
 {
     public string $inputType = 'password';
+
+    /** By setting this variable to true you get an Eye-button on the right side which can toggle the visibility of the entered password */
+    public bool $eye;
+
+    #[\Override]
+    protected function init(): void
+    {
+        parent::init();
+
+        if ($this->eye) {
+            $button = new Button([
+                'class.grey' => true,
+                'iconRight' => 'eye slash',
+            ]);
+
+            $this->action = $button;
+
+            $button->on('click',
+                        new JsExpression("
```

Comment:
```suggestion
            $button->on('click', new JsExpression(<<<'EOF'
                ...
```

prefered coding style